var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -424,7 +424,7 @@ def write_rpath_specs(self):
                               self.prefix.lib, self.prefix.lib64))
         set_install_permissions(specs_file)
 
-    def setup_environment(self, spack_env, run_env):
+    def setup_run_environment(self, spack_env, run_env):
```

Comment:
The new call has been split into two calls. Therefore the signature takes only one environment:
```python
def setup_run_environment(self, env):
```

Reply:
Sorry about that.  Should be fixed in e2f42af.  Let's see what Travis has to say...